libs/jsonforms-components/src/lib/Controls/FormStepper/util/GenerateFormFields.ts

```diff
@@ -1,35 +1,107 @@
-export const resolveLabelFromScope = (scope: string) => {
+import { LabelDescription } from '@jsonforms/core';
+
+type jsonformsLabel = string | boolean | LabelDescription | undefined;
+export const labelToString = (label: jsonformsLabel, scope: string): string => {
+  if (!label) {
+    return resolveLabelFromScope(scope);
+  }
+  if (typeof label === 'object' && label !== null && 'show' in label && label.show) {
```

Comment:
if (typeof label === 'object' && label?.show)  is good enough?

Reply:
No.  On a general object, you first have to test if 'show' is actually in the object, other wise you will get a compile-time error e.g _label?.show_ will have an error something like: "show is not a property of label".  label?.show is a run-time test,  checking to see if label is null before accessing its (known) field.